### PR TITLE
[Bug 17961] docs: Fix error in layerMode dictionary entry

### PR DESCRIPTION
**Pull request by peter-b**

### CODE REVIEW

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 13aeb9a

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 13aeb9ac8e70007fed180b1bff5e281808890649

File: docs/dictionary/property/layerMode.lcdoc [hunk 1]
```diff
@@ -56,8 +56,8 @@ Take a platform game as an example. The main character, any animated elements an
 NON-GAME DRAG EXAMPLE
 Consider a drag-drop interface where a user can 'pick' something up and drop it elsewhere. While the object is being dragged across the stack the <layerMode> can be set to 'dynamic' (particularly important if it has graphic effects applied). Once in its final location the <layerMode> can be returned to 'static'
 
->*Warning:* The <layerMode> only has an effect if the <compositorType property> of the stack is not empty.
+>*Warning:* The <layerMode> only has an effect if the <compositorType> of the stack is not empty.
 
-References: compositorTileSize (property), compositorType property (property), compositorType (property), compositorCacheLimit (property)
+References: compositorTileSize (property), compositorType (property), compositorType (property), compositorCacheLimit (property)
```
Comment (livecodeali) [inline docs/dictionary/property/layerMode.lcdoc]: `compositorType (property)` is in there twice now!

Comment (peter-b) [inline docs/dictionary/property/layerMode.lcdoc]: @livecodeali Derp.  I will fix that.  Does the docs builder get upset by that, by the way?

**Comment by peter-b** [comment]
@livecodeali I've corrected that issue.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 8d823d8

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 8d823d8c9918b9e24705ad4d6a064ebbe42bf9d5

**Comment by livecode-vulcan** [comment]
😎 test success 8d823d8c9918b9e24705ad4d6a064ebbe42bf9d5
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2765)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2774)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2760)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2776)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2782)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2786)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2804)

File: docs/dictionary/property/layerMode.lcdoc (hunk 1 above)
Comment (livecodeali) [inline docs/dictionary/property/layerMode.lcdoc]: @peter-b I don't think so, I suspect it will just overwrite itself

**Pull request merged**